lean4-infoview/src/infoview/main.tsx

```diff
@@ -58,6 +59,8 @@ function Main(props: {}) {
         ret = <p>Waiting for Lean server to start...</p>
     } else if (!curUri) {
         ret = <p>Click somewhere in the Lean file to enable the infoview.</p>
+    } else if (serverStoppedResult){
```

Comment:
I think this else case should come before the `(!curUri)` case.

Reply:
Ok, thanks! fixed